tests/node/rpc/messages/OnVehicleDataTests.js

```diff
@@ -67,11 +69,13 @@ describe('OnVehicleDataTests', function () {
         const testHandsOffSteering = rpcMessage.getHandsOffSteering();
         const testWindowStatus = rpcMessage.getWindowStatus();
         const testGearStatus = rpcMessage.getGearStatus();
+        const testSeatOccupancy = rpcMessage.getSeatOccupancy();
 
         // Valid Tests
         Validator.assertEquals(this.stabilityControlsStatus, testStabilityControlsStatus);
         Validator.assertEquals([Test.GENERAL_WINDOW_STATUS], testWindowStatus);
         Validator.assertEquals(this.gearStatus, testGearStatus);
+        Validator.assertEquals(Test.GENERAL_SEAT_OCCUPANCY, testSeatOccupancy);
```

Comment:
On line 78 are brackets ( [] ) needed around `Test.GENERAL_SEAT_OCCUPANCY` similar to line 76 for `Test.GENERAL_WINDOW_STATUS`?

Reply:
Also not needed, same as in GetVehicleDataResponseTests.js